tools/scripts/acr-build.ps1

```diff
@@ -116,19 +129,24 @@ if ([string]::IsNullOrEmpty($sourceTag)) {
         $sourceTag = "latest"
     }
 }
-
-# Get build root - this is the top most folder with .dockerignore
-$buildRoot = & $getroot -startDir $Path -fileName ".dockerignore"
-# Get meta data
-$metadata = Get-Content -Raw -Path (join-path $Path "container.json") `
-    | ConvertFrom-Json
-
-# Set image name and namespace in acr based on branch and source tag
-$imageName = $metadata.name
-if ($script:Fast.IsPresent) {
-    if (!$metadata.buildAlways) {
-        Write-Host "Using fast build - Skipping optional image $imageName."
-        return
+if (![string]::IsNullOrEmpty($sourceTag)) {
```

Comment:
I cannot see why this has been changed, can you explain?

Reply:
Idea here is to use $prereleaseTag as -rc for builds running on 'main'. Since main should always indicate the code to be released aka it is next release candidate.

Reply:
Explained it offline - in short to generate unique builds to run nightly and CI/CD for every commit. I'll send out the explaining this workflow as well.

Reply:
Cool.  Can we look at using the nuget package version produced by nbgv as tag and then save ourselves all the prerelase tag part?  It would only be a code change in setversion.ps1 and almost none here. Just a suggestion.